test/mock-components/mock-uuidv4.js

```diff
@@ -1,8 +1,8 @@
 module.exports = {
   CORRELATION_ID: '9b232ff2-d0bc-4f0a-993b-3754d66b61b1',
   REFERENCE_ID: '58bac740-35d3-49bd-bc63-bbb0da157bc3',
-  REFERENCE_ID_POST_PAYMENT_ADJUSTMENT: '58bac740-35d3-49bd-bc63-bbb0da157bc4',
-  CORRELATION_ID_POST_PAYMENT_ADJUSTMENT: '9b232ff2-d0bc-4f0a-993b-3754d66b61b2',
-  REFERENCE_ID_SECOND_POST_PAYMENT_ADJUSTMENT: '58bac740-35d3-49bd-bc63-bbb0da157bc5',
-  CORRELATION_ID_SECOND_POST_PAYMENT_ADJUSTMENT: '9b232ff2-d0bc-4f0a-993b-3754d66b61b3'
+  REFERENCE_ID_POST_PAYMENT_ADJUSTMENT: '4f6bfe53-b2b1-4980-b149-8a76fc290464',
+  CORRELATION_ID_POST_PAYMENT_ADJUSTMENT: '853b4483-3625-4f69-9074-01b01407f52b',
+  REFERENCE_ID_SECOND_POST_PAYMENT_ADJUSTMENT: 'e843ad3d-ea01-4f21-9719-8c8790190f49',
+  CORRELATION_ID_SECOND_POST_PAYMENT_ADJUSTMENT: 'cf04031c-31e2-4993-ac8d-98540da194c6'
 }
```

Comment:
I don't understand the need for the change.

Reply:
So this was to make these values clearer for easier detection on which Id is being used during tests as I found it difficult to uncover why there weren't linking properly.

Secondly, and idk if it's true for this case but, these `uuid` are used within our Sequelize db which has its own datatype around it. I believe these values are generated through a function which includes something sim. to a checksum within it. I have seen occasions were Sequelize has rejected a data entry for a `uuid` field because it was not generated and (possibly!) failed some sort of internal check within the value itself.